tensorboard/pip_package/requirements.txt

```diff
@@ -29,5 +29,5 @@ requests >= 2.21.0, < 3
 setuptools >= 41.0.0
 tensorboard-data-server >= 0.6.0, < 0.7.0
 tensorboard-plugin-wit >= 1.6.0
-werkzeug >= 0.11.15, < 2.1.0
+werkzeug >= 0.11.15
```

Comment:
While we're modifying this, IMO we might as well update the requirement to `werkzeug >= 1.0.1`. That's the version that internal third_party werkzeug is using, which is generally nice to align with to avoid unexpected feature gaps.  The 0.11.15 version is very old (from 2016, compared to 2020 for 1.0.1) and I don't know of any reason we should still support it.

Reply:
Thanks! Updated.
For testing the internal sync, it passed yesterday but not now because of some other PR got in since that.